Makefile

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+HOST_MACHINE    = $(shell uname -m)
+CONTAINER_TAG   = nsm-api
+DOCKERFILE_PATH = Dockerfiles/Dockerfile.build
+COMP_VERSION    = 1.40
+STABLE          = stable
+NIGHTLY         = nightly
+
+.build-${HOST_MACHINE}-${COMP_VERSION}: ${DOCKERFILE_PATH}
```

Comment:
I think you can just add an `all` target before this, instead of adding `.something` build rules. This just means that they can't be the default build rule, and I'm not sure how important that is.

Reply:
The build rules are for building the docker images. Merging them all into an all build rule, would not allow the option to build them independently.